homeworks/lesson2/app/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react';
+import {BrowserRouter as Router, Route, Link, Switch} from 'react-router-dom';
+import {Jumbotron} from 'react-bootstrap';
+import {LoginForm, CourseList, EditCourse, Page404} from './components/index';
+import './App.css';
+
+class App extends Component {
+    render() {
+        return (
+            <div className="App">
+                <Jumbotron>
+                    <h2>Application for React course.</h2>
+                    <p>
+                        Oleksii Popov
+                    </p>
+                </Jumbotron>
+
+                <Router>
```

Comment:
Please move that to another component.
Because that is another logic and not view and should be located not in a presentational layer.

Reply:
Updated